src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -34,9 +34,9 @@ __PORTABLETARGETS_PROJECT_CONTENT="
 
 __PUBLISH_TFM=netcoreapp2.0
 
-__DEFAULT_RESTORE_ARGS="--no-cache --packages \"${__PACKAGES_DIR}\""
-__INIT_TOOLS_RESTORE_ARGS="${__DEFAULT_RESTORE_ARGS} --source https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json --source https://api.nuget.org/v3/index.json ${__INIT_TOOLS_RESTORE_ARGS:-}"
```

Comment:
>__INIT_TOOLS_RESTORE_ARGS [](start = 172, length = 25)

It looks like previously someone could have defined `__INIT_TOOLS_RESTORE_ARGS` before this script and the previous value would get used in this variable. But now that behavior looks no longer possible. Is that OK?

Reply:
Plus, we could have a variable for the sources, that one doesn't need path evaluation does it?

Reply:
No the sources variable doesnot need any path evaluation.

> `__INIT_TOOLS_RESTORE_ARGS` before this script and the previous value would get used in this variable. But now that behavior looks no longer possible

i will add that back